lms/djangoapps/verify_student/signals.py

```diff
@@ -75,3 +75,9 @@ def send_idv_update(sender, instance, **kwargs):  # pylint: disable=unused-argum
         photo_id_name=instance.name,
         full_name=full_name
     )
+
+
+@receiver(USER_RETIRE_LMS_MISC)
+def _listen_for_lms_retire_verification_attempts(sender, **kwargs):  # pylint: disable=unused-argument
+    user = kwargs.get('user')
+    VerificationAttempt.retire_user(user.id)
```

Comment:
Should we handle exception during deletion? I feel like we should at least log info here to let the logger know we are deleting users.

Reply:
I had this same thought as you on Alie's `persona-integration` pull request: https://github.com/edx/persona-integration/pull/17, but my final opinion was that because this code runs as part of a separate retirement job that runs the retirement scripts, it would probably be better for it to fail loudly than to fail silently so that we know something went wrong. My thinking was that no one is going to be reviewing the output of that job to look for silent failures, but loud failures may get someone's attention.